services/app/src/components/pageLayout/DarkLayout.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './layout.css';
+import Header from '../header/Header';
+import Footer from '../pageFooter/Footer';
+
+const Layout = ({ children }) => (
```

Comment:
Component name should match the file name, but the file name should be something different, as dark layout implies dark theme.